### PR TITLE
Added styling

### PR DESCRIPTION
**Pull request by MarkvanAlphen**

### CODE REVIEW

**Comment by MarkvanAlphen** [comment]
Added media query for the standings table

**Comment by rjbijl** [comment]
Please revert the permissions on the gitignore files

File: app/Resources/views/base.html.twig [hunk 1]
```diff
@@ -2,10 +2,10 @@
 <html>
     <head>
         <meta charset="UTF-8" />
-        <title>{% block title %}Welcome!{% endblock %}</title>
+        <title>{% block title %}Prezent Foosball competition{% endblock %}</title>
```
Comment (rjbijl) [inline app/Resources/views/base.html.twig]: I want to keep this as generic as possible, so remove Prezent here. Will add variable to parameter.yml for site name later on

**Comment by rjbijl** [comment]
tnx

**Pull request merged**